### PR TITLE
Adds email style

### PR DESCRIPTION
**Pull request by paoloromolini**
2018-08-07-124045_1476x654_scrot

2018-08-14-121011_944x668_scrot

Adds email template and css

### CODE REVIEW

**Review by yakky — COMMENTED** [review]
maybe we could use something like premailer to keep css and image in external files and compile into the email message only when the message is sent?

**Comment by paoloromolini** [comment]
Sounds good, I've never done it though. Are you referring to something like that https://github.com/alexhayes/django-premailer ?

**Comment by yakky** [comment]
Or https://github.com/peterbe/premailer (which we already use, you can ping @protoroto which has implemented it recently)

File: docs_italia_convertitore_web/views.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def form_valid(self, form):
             for chunk in uploaded.chunks():
                 destination.write(chunk)
         options_json = form.get_options_json(new_folder_name)
-        process_file.delay(email, saved, unique_key, options_json=options_json)
+        process_file(email, saved, unique_key, options_json=options_json)
```
Comment (yakky) [inline docs_italia_convertitore_web/views.py]: why removing this?

File: requirements_test.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ mock>=1.0.1
 flake8>=2.1.0
 tox>=1.7.0
 codecov>=2.0.0
+premailer
```
Comment (yakky) [inline requirements_test.txt]: We only need dependency in the main requirements file

File: requirements_dev.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
 bumpversion==0.5.3
+premailer
```
Comment (yakky) [inline requirements_dev.txt]: We only need dependency in the main requirements file

**Review by yakky — CHANGES_REQUESTED** [review]

**Review by yakky — APPROVED** [review]

**Review by francescozaia — APPROVED** [review]

**Pull request merged**